components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/PolicyOperationDAOImpl.java

```diff
@@ -39,6 +39,8 @@
     public int addOperation(Operation operation) throws OperationManagementDAOException {
         int operationId;
         PreparedStatement stmt = null;
+        ByteArrayOutputStream bao = null;
+        ObjectOutputStream oos = null;
         try {
```

Comment:
Cant we use try with resource here?

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275